package.json

```diff
@@ -9,7 +9,9 @@
     "test": "jest",
     "coverage": "jest --coverage",
     "seed-db": "node seed.js",
-    "build": "./node_modules/.bin/webpack --config ./webpack.prod.config.js"
+    "build": "./node_modules/.bin/webpack --config ./webpack.config.js",
+    "create-json": "node system-design/data-generator.js",
```

Comment:
you could probably combine this with the mongo-import script by just putting a && between them.